keyboards/handwired/franky36/keyboard.json

```diff
@@ -0,0 +1,77 @@
+{
+    "manufacturer": "Grigory Avdyushin",
+    "keyboard_name": "franky36",
+    "maintainer": "Grigory Avdyushin",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "oled": true
```

Comment:
If OLEDs are enabled at the keyboard level, there needs to exist some configuration for them at the keyboard level in a `frank36.c` file.